File: windows-client/ComponentVault.WinUI/Models/ComponentDraft.cs
namespace ComponentVault.WinUI.Models;

public sealed class ComponentDraft
{
    public string? Id { get; init; }
    public required string Sku { get; init; }
    public required string Name { get; init; }
    public required string Category { get; init; }
    public required string PackageName { get; init; }
    public required string Location { get; init; }
    public string Description { get; init; } = string.Empty;
    public required int Quantity { get; init; }
    public required int MinStock { get; init; }
}

File: android-client/app/src/main/java/com/componentvault/android/model/ComponentDraft.kt
package com.componentvault.android.model

data class ComponentDraft(
    val id: String? = null,
    val sku: String,
    val name: String,
    val category: String,
    val packageName: String,
    val location: String,
    val description: String,
    val quantity: Int,
    val minStock: Int,
)
